app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd, source_file, second_file = command.split(" ")
```

Comment:
`" "` is the default separator.
https://www.w3schools.com/python/ref_string_split.asp